src/Microsoft.Extensions.Configuration.UserSecrets/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Value cannot be null or an empty string.</value>
   </data>
   <data name="Error_EntryAssemblyNull" xml:space="preserve">
-    <value>This method could not find a user secret ID because the application's entry assembly is not set. Try using the ".AddUserSecrets(Assembly assembly)" method instead.</value>
+    <value>This method could not find a user secret ID because the application's entry assembly is not set. Try using the ".AddUserSecrets&lt;T&gt;()" or ".AddUserSecret(string userSecretsId)" method instead.</value>
```

Comment:
Not sure I understand. Do you mean we should also include the implied "this" parameter?
`AddUserSecrets<T>(this IConfigurationBuilder configuration)`